src/SDKs/_metadata/containerregistry_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2017-10-12 19:25:10 UTC
+2018-07-31 07:12:56 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: Azure
-Branch:      current
-Commit:      55813058b0105edaf813acaaf734487fcdc8e46d
+GitHub user: northtyphoon
+Branch:      acrbuild
```

Comment:
@northtyphoon we will not allow to publish SDKs that are not being generated from the master branch.
So you will have to generate it from the master branch of the rest spec repo.

Reply:
Updated

---
In reply to: [207694451](https://github.com/Azure/azure-sdk-for-net/pull/4630#discussion_r207694451) [](ancestors = 207694451)